source/extensions/omni.isaac.lab/omni/isaac/lab/sensors/camera/camera.py

```diff
@@ -541,17 +540,21 @@ def _update_intrinsic_matrices(self, env_ids: Sequence[int]):
             # get camera parameters
             focal_length = sensor_prim.GetFocalLengthAttr().Get()
             horiz_aperture = sensor_prim.GetHorizontalApertureAttr().Get()
+            vert_aperture = sensor_prim.GetVerticalApertureAttr().Get()
+            horiz_aperture_offset = sensor_prim.GetHorizontalApertureOffsetAttr().Get()
+            vert_aperture_offset = sensor_prim.GetVerticalApertureOffsetAttr().Get()
```

Comment:
Do these now work properly from rendering PoV? I don't know if this issue is resolved from Omniverse side. Also vertical and horizontal aperture should be same for "square" pixels?

Reply:
I don't know if they work properly. Will implement a test to see if the results between between raycaster and USD camera are the same if I set them

Reply:
apparently it is not resolved

Reply:
offesets are not resolved

Reply:
From what I remember the vertical aperture doesn't actually change the rendering of the image. Ultimately it comes down to the resolutions and horizontal aperture. The vertical aperture does change the visualized frustum though.